### ISSUE TITLE
Modals

### ISSUE THREAD

**Issue by schalkneethling** [mdn/yari#1179]
Inside Yari modals are based on [react-modal](http://reactcommunity.org/react-modal/accessibility/). Ensure that all relevant styling is moved in Yari and that all functionality still works as before.

Ensure that we are using the modal appropriately so that we benefit from all built-in accessibility features. If some features are missing, document and see how we can contribute back and in so doing, benefit inside MDN

**Comment by schalkneethling** [comment]
Resolved by https://github.com/mdn/yari/pull/1537
